src/test/java/org/jnode/fs/DataStructureAsserts.java

```diff
@@ -19,8 +21,8 @@ public static void assertStructure(FileSystem<?> fileSystem, String expected) th
         StringBuilder actual = new StringBuilder(expected.length());
 
         actual.append(String.format("type: %s vol:%s total:%d free:%d\n",
-                                    fileSystem.getType().getName(), fileSystem.getVolumeName(),
-                                    fileSystem.getTotalSpace(), fileSystem.getFreeSpace()));
+                fileSystem.getType().getName(), fileSystem.getVolumeName().replaceAll("\0", ""),
```

Comment:
```suggestion
                fileSystem.getType().getName(), fileSystem.getVolumeName(),
```
Let's not rely on the unit test to format the output of the `getVolumeName()` function.

Reply:
The suggested change was applied,  thanks,  The data is formatted in the source that is thee superblock class